src/Controller/OcrController.php

```diff
@@ -60,6 +60,7 @@ class OcrController extends AbstractController {
 		'langs' => [],
 		'psm' => TesseractEngine::DEFAULT_PSM,
 		'crop' => [],
+		'lineId' => TranskribusEngine::DEFAULT_LINEID
```

Comment:
Nitpick: it doesn't matter at all, but I think URL & API component names should either be camelCase or snake_case, but not both. It looks like we've got more of the latter so far (although most are single words so aren't either), so I suggest `line_id` here.